scripts/semconv/changelog-gen.js

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env node
+/**
+ * A script to generate a meaningful changelog entry for an update in
+ * semantic conventions version.
+ *
+ * Usage:
+ *    vi scripts/semconv/generate.sh  # Typically update SPEC_VERSION to latest.
+ *    ./scripts/semconv/generate.sh   # Re-generate the semconv package exports.
+ *    ./scripts/semconv/changelog-gen.sh
```

Comment:
wow! 

JFYI: you might be interested in https://github.com/open-telemetry/weaver/issues/432 - we're working on the tooling that would be able to generate yaml diff between two versions of semconv (the goal is to to enable transformations, generate migration guides, changelogs, etc) .

Reply:
Thanks for the pointer.